src/css/style.css

```diff
@@ -36,54 +36,57 @@ body {
 .header {
   display: flex;
   justify-content: space-between;
-  padding: 0 6.25rem;
-  margin-top: 25px;
+  width: 90%;
+  margin: 0 auto;
+  padding-top: 2.8rem;
 }
 .logo img {
-  width: 140px;
-  height: 120px;
+  width: 230px;
+  object-fit: cover;
 }
 .language {
+  height: 100%;
   display: flex;
+  gap: 1rem;
   align-items: center;
 }
-.world {
-  display: flex;
-  align-items: center;
-  margin-right: 10px;
+.world img {
+  object-fit: contain;
 }
-.select-language {
-  display: flex;
-  align-items: center;
+.arrow-bottom {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  background-image: url('../assets/Vector.png');
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-size: 12px;
 }
 .content-select select {
+  padding-right: 1.8rem;
+  padding-left: 1.8rem;
   width: 203px;
   height: 44px;
```

Comment:
Would be good to have this in Rem, maybe another similar cases

Reply:
I converted everything except small units. like borders-width

Reply:
Yes, border width and radius are fine. Thanks